apps/src/applab/designElements/button.jsx

```diff
@@ -208,41 +209,70 @@ export default {
   themeValues: {
     backgroundColor: {
       type: 'color',
+      default: color.applab_default_button_background_color,
       classic: color.applab_button_teal,
-      dark: color.yellow
+      orange: color.applab_orange_button_background_color,
+      citrus: color.applab_citrus_button_background_color
     },
     borderRadius: {
+      default: 4,
       classic: 0,
-      dark: 10
+      orange: 0,
+      citrus: 2
     },
     borderWidth: {
+      default: 1,
       classic: 0,
-      dark: 0
+      orange: 2,
+      citrus: 2
     },
     borderColor: {
       type: 'color',
+      default: color.applab_default_button_border_color,
       classic: color.black,
-      dark: color.white
+      orange: color.applab_orange_button_border_color,
+      citrus: color.applab_citrus_button_border_color
     },
     textColor: {
       type: 'color',
+      default: color.applab_default_button_text_color,
       classic: color.white,
-      dark: color.black
+      orange: color.applab_orange_text_color,
+      citrus: color.applab_citrus_text_color
+    },
+    fontFamily: {
+      default: 'Arial Black',
+      classic: 'Arial',
+      orange: 'Verdana',
+      citrus: 'Georgia'
+    },
+    fontSize: {
+      default: 18,
+      classic: 14,
+      orange: 18,
+      citrus: 18
     }
   },
   create: function() {
     const element = document.createElement('button');
     element.appendChild(document.createTextNode('Button'));
     element.style.padding = '0px';
     element.style.margin = '0px';
-    element.style.height = '30px';
-    element.style.width = '80px';
-    element.style.fontFamily = fontFamilyStyles[0];
-    element.style.fontSize = defaultFontSizeStyle;
     if (experiments.isEnabled('applabThemes')) {
       element.style.borderStyle = 'solid';
+      // Roughly scale default size based on the current theme's font size:
+      const currentTheme = elementLibrary.getCurrentTheme(
+        designMode.activeScreen()
+      );
+      const fontSize = this.themeValues.fontSize[currentTheme];
+      element.style.height = `${growToGridSize(fontSize * 2)}px`;
+      element.style.width = `${10 + growToGridSize(fontSize * 5)}px`;
```

Comment:
Just making sure - this only happens when we create the element, not when changing themes, correct?  Does manually setting these styles mess with resize-as-you-type behavior for elements with default sizes at all?

Reply:
This is only for the button. There is no resize-as-you-type behavior on that element. Previously, all buttons created in design mode were hardcoded as 80x30 pixels. That will still be the case if the screen is in the "Classic" theme. However, they may be larger if created when the screen is in one of the other new themes.